File: C/sum_of_digits_of_number.c
#include <stdio.h>

int main() {
    int number, digit, sum_digits = 0;

    printf("Enter a number: ");
    scanf("%d", &number);

    while (number > 0) {
        digit = number % 10;
        sum_digits = sum_digits + digit;
        number = number / 10;
    }

    printf("The sum of digits is: %d\n", sum_digits);

    return 0;
}

File: Python/sum_of_digits_of_number.py
number = int(input("Enter a number:"))
number=abs(number)
sum_digits = 0
print(f"The number is: {number}")
while number>0:
    digit = number%10
    sum_digits = sum_digits + digit
    number = number//10
print(f"The sum of digits of number is: {sum_digits}")